app/src/main/java/org/merakilearn/EnrollActivity.kt

```diff
@@ -153,6 +153,19 @@ class EnrollActivity : AppCompatActivity() {
             enroll.isVisible = false
         }
 
+        it.language?.let {
+            tvClassLanguage.isVisible = true
+            if(it.toLowerCase().equals("hi")) {
+                tvClassLanguage.text = "HINDI"
+            }else if(it.toLowerCase().equals("en")) {
+                tvClassLanguage.text = "ENGLISH"
+            }else{
+                tvClassLanguage.text ="OTHER"
+            }
+        } ?: kotlin.runCatching {
+            tvClassLanguage.isVisible = false
```

Comment:
Hardcoding is not such a good idea, Maybe we can fetch this from Android via Locale

Reply:
runCatching is also not needed here

Reply:
Done